apps/judicial-system/web/src/utils/hooks/useDefendants/index.ts

```diff
@@ -78,6 +84,26 @@ const useDefendants = () => {
     [formatMessage, updateDefendantMutation],
   )
 
+  // TODO: This is maybe not ideal?
+  const limitedAccessUpdateDefendant = useCallback(
```

Comment:
This is a bit redundant of `updateDefendant` above. In cases we handle these the mutations calls within the same function by checking the user...but maybe this is ok for now?

Reply:
Yeah I feel like for this specific case it's fine for it to be like this now. The reason for the handling in the case mutations was because we have some shared components between prosecutors and defenders so instead of needing to always have an if else inside each component to decide which mutation to use we made the single hook that checks for us. 
Since right now this specific case is still really limited (haha) to this single use I'd say its fine to have them separate and we can then just combine them later if needed 🙌